src/core_modules/capture-core/metaDataMemoryStoreBuilders/programs/factory/transformNodeFuntions/transformNodeFunctions.js

```diff
@@ -0,0 +1,58 @@
+// @flow
+import React from 'react';
+
+export const transformTrackerNode = (node: Object, index: number, nodeToElementFn: (...args : Array<any>) => any) => {
+    if (node.name === 'input') {
+        const htmlElementId = node.attribs?.attributeid;
+
+        if (htmlElementId) {
+            const inputElement = nodeToElementFn(node, index);
+
+            const style = inputElement.props?.style;
+            const className = inputElement.props?.className;
+
+            const customFormElementProps = {
+                id: htmlElementId,
+                style,
+                className,
+            };
+
+            return React.createElement(
+                'FormField', {
+                    customFormElementProps,
+                    id: htmlElementId,
+                },
+            );
+        }
+        return null;
+    }
+    return undefined;
+};
+
+export const transformEventNode = (node: Object, index: number, nodeToElementFn: (...args : Array<any>) => any) => {
+    if (node.name === 'input') {
+        const htmlElementId = node.attribs && node.attribs.id;
+        const matchResult = htmlElementId && /-[^-]+/.exec(htmlElementId);
```

Comment:
Hey, 
I think that having the `/-[^-]+/`regex stored in a const with a descriptive name can improve the code readability. WDYT? Thanks!

Reply:
Hey,
Absolutely agree, implemented!
Thanks